environments.json

```diff
@@ -936,7 +936,7 @@
     "short": "FF 52 ESR",
     "release": "2017-03-07",
     "retire": "2018-09-05",
-    "obsolete": false
+    "obsolete": "very"
```

Comment:
This should be true, not "very".

Reply:
Not sure about that. I believe that extended support end of life makes this version _very_ obsolete

Reply:
Well, it seems everyone has an opinion on this subject (see #740).  What I have been using so far is one year after the support ended.

Reply:
ESR is just security updates. Firefox 52 retire date is 2017-04-19 so it's really very obsolete version

Reply:
imo it's not hardly obsolete at all until the next security update comes out, that this one doesn't include.

Reply:
Last security update was on June 26 (Firefox 52.9) 
https://wiki.mozilla.org/Release_Management/Calendar
That ESR version reached it’s EOL 
https://twitter.com/fxsitecompat/status/1037369380869263360

Reply:
I still insist that Firefox 52 is _very_ obsolete version. It’s release was years ago

Reply:
I can change this line to show `false` (I'm still prefer it to be `"very"` though) to just reach any consensus here. We should formalize definitions of retirement, obsoleteness or whatever affects presence of environments in tables under checkboxes or not because it becomes really difficult to reach consensus.

Reply:
Ok, I did it (it's wrong though)